### ISSUE TITLE
Upgrades

### ISSUE THREAD

**Issue by gezquinndesign** [RevenueCat/purchases-js#182]
Is there any way to handle upgrades?

**Comment by nicfix** [comment]
Hi @gezquinndesign, thanks for trying `purchases-js`!

If by upgrades you mean upgrading one customer from one plan to another we don't have that yet supported through `purchases-js`. I'll bring back this requirement to the team!

**Comment by gezquinndesign** [comment]
Yeah, that's what I mean.  Essentially, all the other platforms have [something similar](https://www.revenuecat.com/docs/subscription-guidance/managing-subscriptions).  I can't see any graceful way of managing upgrades in the backend or via the customer portal either.  Thanks for passing the requirement on, would be nice to have this functionality before we release it out in the wild.

**Comment by alfondotnet** [comment]
@gezquinndesign, bit late, but we are currently working on this feature, will leave the issue open until we release
